src/js/view/controls/components/timeslider.ts

```diff
@@ -293,7 +322,7 @@ class TimeSlider extends Slider {
         }
     }
 
-    updateAriaText() {
+    updateAriaText(): void {
         const model = this._model;
         if (model.get('seeking')) {
```

Comment:
This is never set. 'seeking' is not a model property. I confirmed by searching the player projects and checking `jwplayer().getConfig()`.

Reply:
We already check `scrubbing` prior to calling `updateAriaText()`